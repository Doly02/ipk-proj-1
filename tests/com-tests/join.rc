/auth test1 PASSWORD MyDisplayName
/join general