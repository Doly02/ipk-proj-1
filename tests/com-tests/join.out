/auth test1 PASSWORD MyDisplayName
Success: Authentication successful.
Server: MyDisplayName joined discord.general.
Server: MyDisplayName joined general.
Success: Channel general successfully joined.